sql/api/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -8073,6 +8161,16 @@ object functions {
   def make_interval(years: Column): Column =
     Column.fn("make_interval", years)
 
+  /**
+   * This is a special version of `make_interval` that performs the same operation, but returns a
+   * NULL value instead of raising an error if interval cannot be created.
+   *
+   * @group url_funcs
+   * @since 4.0.0
+   */
+  def try_make_interval(): Column =
```

Comment:
Why do you need this one? In which cases does `make_interval()` fail?

Reply:
You are right, we cannot get an error here, as this is the same as makeinterval(0) which is not an overflow. Will update the pr.